StreetPatch.Data/Entities/SignUpData.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace StreetPatch.Data.Entities
```

Comment:
The class itself is okay. However, I would suggest moving it to a new folder - for example: Entities/DTO. The reason for that is because this is not a class that will represent a table in the database, but rather - some form of input parsing.